packages/mux-player/src/template.ts

```diff
@@ -114,8 +109,31 @@ export const ChromeRenderer = (props: MuxTemplateProps) => {
 };
 
 // prettier-ignore
-const MediaPlayButton = () => html`
-  <media-play-button>
+const SeekToLiveButton = (props: Partial<MuxTemplateProps>) => html`
+  <button 
+    aria-disabled="${props.inLiveWindow}"
+    in-live-window="${props.inLiveWindow}"
+    onclick="${function (this: HTMLButtonElement, evt: Event) {
```

Comment:
doing this inline, but that means a new function will be generated whenever a (re)render occurs. We can "promote" this to props/state if this is a concern.

Reply:
why does `this` need to be a function argument? isn't it accessible anyway via this

Reply:
why does `this` need to be a function argument? isn't it accessible anyway via this

Reply:
short version: "because typescript"